### PR TITLE
RC-295 Refactored components in website settings modal

### PR DESCRIPTION
**Pull request by ryaaoka**
[Ticket ](https://dbmedialab.atlassian.net/browse/RC-295)

### CODE REVIEW

File: src/admin/components/website/modalParts/WebsiteEditors.tsx [hunk 1]
```diff
@@ -19,14 +19,17 @@
 import * as React from 'react';
 import * as UsersActions from 'admin/actions/UsersActions';
 
+import { LabeledSelect } from '../additionalComponents/LabeledSelect';
```
Comment (GrimStal) [inline src/admin/components/website/modalParts/WebsiteEditors.tsx]: Preferable full path

File: src/admin/components/website/modalParts/WebsiteHosts.tsx [hunk 2]
```diff
@@ -108,32 +102,24 @@ class WebsiteHosts extends React.Component <any, any> {
...
+							</span>}
+						ID={`hosts-link`}
 					/>
-					<InputError
-						errorText={this.checkExistingHosts()}
-						touchedField={this.state.touched}
+					<TagList
+						items={this.props.hosts}
+						onDelete={this.onDelete}
+						classes={'website-hosts-list'}
```
Comment (GrimStal) [inline src/admin/components/website/modalParts/WebsiteHosts.tsx]: What about css refactoring for similar working components?

File: src/admin/components/website/additionalComponents/LabeledSelect.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+export interface IOption {
+	value: string;
+	name: string;
+}
+
+export class LabeledSelect extends React.Component <ILabeledSelect, any> {
+
+	constructor (props) {
```
Comment (DonikaV) [inline src/admin/components/website/additionalComponents/LabeledSelect.tsx]:
```tsx
constructor (props: ILabeledSelect) {
    
}

File: src/admin/components/website/additionalComponents/Label.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+import * as React from 'react';
+
+export interface ILabel {
+	label: JSX.Element | string,
+	ID: string;
+}
+
+export class Label extends React.Component <ILabel, any> {
+	constructor (props) {
```
Comment (DonikaV) [inline src/admin/components/website/additionalComponents/Label.tsx]: constructor (props: ILabel) {
    
}

File: src/admin/components/website/additionalComponents/TagList.tsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+export interface ITagList {
+	classes: string;
+	onDelete: (index: number) => void;
+	items: string[];
+}
+
+export class TagList extends React.Component <ITagList, any> {
+
+	constructor (props) {
```
Comment (DonikaV) [inline src/admin/components/website/additionalComponents/TagList.tsx]:
```tsx
constructor (props: ITagList) {
    
}

**Pull request merged**